src/main/java/com/google/devtools/build/lib/rules/cpp/CcModule.java

```diff
@@ -1432,7 +1432,7 @@ public boolean isCcToolchainResolutionEnabled(SkylarkRuleContext skylarkRuleCont
                     .getActionConstructionContext()
                     .getConfiguration()
                     .getFragment(CppConfiguration.class),
-                ((BazelStarlarkContext) starlarkContext).getSymbolGenerator())
+                ((BazelStarlarkContext) starlarkContext).getSymbolGenerator(), null)
```

Comment:
/* ruleContext= */ null. Also in other places.

Reply:
Why is null here ok? You can get to action construction context using actions.getActionConstructionContext() there.

Reply:
I'll check that!